cmd/fleetctl/debug.go

```diff
@@ -92,7 +92,7 @@ func debugProfileCommand() *cli.Command {
 
 			outfile := getOutfile(c)
 			if outfile == "" {
-				outfile = outfileName("profile")
+				outfile = outfileNameWithExt("profile", "pb")
```

Comment:
nitpick, use `.prof` instead of `.pb` as the extension. `.pb` is commonly used for protobuf files.

Reply:
great idea, thanks! I also abstracted all the strings into a `const`.